### DISCUSSION TITLE
2️⃣ 1차 스터디를 위한 프로그래머스 Level 2 문제 선택

### DISCUSSION THREAD

**Discussion by hyeinisfree** [hyeinisfree/kings-algorithm-study#2] [문제 추천하기]
## 📚 1차 스터디를 위한 프로그래머스 Level 2 문제 선택
- 1차 스터디 기간 : 2022.02.07 ~ 2022.03.07
- week3, 4 : 프로그래머스 Level 2 문제 풀이
- 2022.02.02까지 스터디원 모두 Level 2 문제를 10문제씩 추천
- 코멘트 반응을 통해 최종 문제를 선택

**Comment by hyeinisfree** [comment]
- 분류 : 프로그래머스 Level 2
- 문제 : [멀쩡한 사각형](https://programmers.co.kr/learn/courses/30/lessons/62048)
- 추천인 : 김혜인

**Comment by hyeinisfree** [comment]
- 분류 : 프로그래머스 Level 2
- 문제 : [기능개발](https://programmers.co.kr/learn/courses/30/lessons/42586)
- 추천인 : @hyeinisfree @chaewss

**Comment by hyeinisfree** [comment]
- 분류 : 프로그래머스 Level 2
- 문제 : [타겟 넘버](https://programmers.co.kr/learn/courses/30/lessons/43165)
- 추천인 : @hyeinisfree @sio2whocodes

**Comment by hyeinisfree** [comment]
- 분류 : 프로그래머스 Level 2
- 문제 : [행렬 테두리 회전하기](https://programmers.co.kr/learn/courses/30/lessons/77485)
- 추천인 : @hyeinisfree @sio2whocodes @milsakim

**Comment by hyeinisfree** [comment]
- 분류 : 프로그래머스 Level 2
- 문제 : [빛의 경로 사이클](https://programmers.co.kr/learn/courses/30/lessons/86052)
- 추천인 : @hyeinisfree @milsakim

**Comment by hyeinisfree** [comment]
- 분류 : 프로그래머스 Level 2
- 문제 : [조이스틱](https://programmers.co.kr/learn/courses/30/lessons/42860)
- 추천인 : @hyeinisfree @sio2whocodes @milsakim

**Comment by hyeinisfree** [comment]
- 분류 : 프로그래머스 Level 2
- 문제 : [괄호 회전하기](https://programmers.co.kr/learn/courses/30/lessons/76502)
- 추천인 : @hyeinisfree @sio2whocodes @milsakim

**Comment by hyeinisfree** [comment]
- 분류 : 프로그래머스 Level 2
- 문제 : [위장](https://programmers.co.kr/learn/courses/30/lessons/42578)
- 추천인 : @hyeinisfree @chaewss @sio2whocodes

**Comment by hyeinisfree** [comment]
- 분류 : 프로그래머스 Level 2
- 문제 : [점프와 순간 이동](https://programmers.co.kr/learn/courses/30/lessons/12980)
- 추천인 : 김혜인

**Comment by hyeinisfree** [comment]
- 분류 : 프로그래머스 Level 2
- 문제 : [n^2 배열 자르기](https://programmers.co.kr/learn/courses/30/lessons/87390)
- 추천인 : 김혜인

**Comment by chaewss** [comment]
- 분류 : 프로그래머스 Level 2
- 문제 : [큰 수 만들기](https://programmers.co.kr/learn/courses/30/lessons/42883)
- 추천인 : @chaewss @sio2whocodes

**Comment by chaewss** [comment]
- 분류 : 프로그래머스 Level 2
- 문제 : [다리를 지나는 트럭](https://programmers.co.kr/learn/courses/30/lessons/42583)
- 추천인 : 서채원

**Comment by chaewss** [comment]
- 분류 : 프로그래머스 Level 2
- 문제 : [소수 찾기](https://programmers.co.kr/learn/courses/30/lessons/42839)
- 추천인 : @chaewss @milsakim

**Comment by chaewss** [comment]
- 분류 : 프로그래머스 Level 2
- 문제 : [카펫](https://programmers.co.kr/learn/courses/30/lessons/42842)
- 추천인 : 서채원

**Comment by chaewss** [comment]
- 분류 : 프로그래머스 Level 2
- 문제 : [최솟값 만들기](https://programmers.co.kr/learn/courses/30/lessons/12941)
- 추천인 : 서채원

**Comment by chaewss** [comment]
- 분류 : 프로그래머스 Level 2
- 문제 : [스킬트리](https://programmers.co.kr/learn/courses/30/lessons/49993)
- 추천인 : 서채원

**Comment by chaewss** [comment]
- 분류 : 프로그래머스 Level 2
- 문제 : [올바른 괄호](https://programmers.co.kr/learn/courses/30/lessons/12909)
- 추천인 : 서채원

**Comment by chaewss** [comment]
- 분류 : 프로그래머스 Level 2
- 문제 : [N개의 최소공배수](https://programmers.co.kr/learn/courses/30/lessons/12953)
- 추천인 : 서채원

**Comment by sio2whocodes** [comment]
- 분류 : 프로그래머스 Level 2
- 문제 : [프린터](https://programmers.co.kr/learn/courses/30/lessons/42587)
- 추천인 : @sio2whocodes

**Comment by sio2whocodes** [comment]
- 분류 : 프로그래머스 Level 2
- 문제 : [가장 큰 수](https://programmers.co.kr/learn/courses/30/lessons/42746)
- 추천인 : @sio2whocodes

**Comment by sio2whocodes** [comment]
- 분류 : 프로그래머스 Level 2
- 문제 : [쿼드압축 후 개수 세기](https://programmers.co.kr/learn/courses/30/lessons/68936)
- 추천인 : @sio2whocodes

**Comment by sio2whocodes** [comment]
- 분류 : 프로그래머스 Level 2
- 문제 : [방문 길이](https://programmers.co.kr/learn/courses/30/lessons/49994)
- 추천인 : @sio2whocodes

**Comment by milsakim** [comment]
- 분류 : 프로그래머스 Level 2
- 문제 : [짝지어 제거하기](https://programmers.co.kr/learn/courses/30/lessons/12973)
- 추천인 : @milsakim

**Comment by milsakim** [comment]
- 분류 : 프로그래머스 Level 2
- 문제 : [삼각 달팽이](https://programmers.co.kr/learn/courses/30/lessons/68645)
- 추천인 : @milsakim

**Comment by milsakim** [comment]
- 분류 : 프로그래머스 Level 2
- 문제 : [교점에 별 만들기](https://programmers.co.kr/learn/courses/30/lessons/87377)
- 추천인 : @milsakim

**Comment by imim-seung** [comment]
분류 : 프로그래머스 Level 2
문제 : 더 맵게
추천인 : 김승미

**Comment by imim-seung** [comment]
분류 : 프로그래머스 Level 2
문제 : 2개 이하로 다른 비트
추천인 : 김승미

**Comment by imim-seung** [comment]
분류 : 프로그래머스 Level 2
문제 : 구명보트
추천인 : 김승미

**Comment by imim-seung** [comment]
분류 : 프로그래머스 Level 2
문제 : 땅따먹기
추천인 : 김승미
